docs/changelog.rst

```diff
@@ -83,6 +88,11 @@ Gapless
 - Tests have been updated to always use a core actor so async state changes
   don't trip us up.
 
+- Seek events are now triggered when the seek completes. Further changes have
```

Comment:
Sounds like we didn't trigger that event at all before. Need to explain what the old behavior was.